contracts/governance/AxelarMultisigMintLimiter.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { SafeNativeTransfer } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/utils/SafeTransfer.sol';
+import { IMultisigExecute } from '../interfaces/IMultisigExecute.sol';
+import { MultisigBase } from '../auth/MultisigBase.sol';
+
+contract AxelarMultisigMintLimiter is MultisigBase, IMultisigExecute {
+    using SafeNativeTransfer for address;
+
+    constructor(address[] memory accounts, uint256 threshold) {
+        _rotateSigners(accounts, threshold);
+    }
+
+    function execute(
+        address target,
+        bytes calldata callData,
+        uint256 nativeValue
+    ) external payable onlySigners {
+        if (msg.value < nativeValue) revert InsufficientValue();
```

Comment:
Similar to the governance contract, I think we should just assume the value is present on the contract's balance already, rather than one specific multisig participant forwarding it. It removes some of the complexity below